community/kernel/src/main/java/org/neo4j/kernel/impl/store/StoreFactory.java

```diff
@@ -123,16 +119,47 @@ public void setPageCache( PageCache pageCache )
         this.pageCache = pageCache;
     }
 
-    public NeoStores openNeoStoresEagerly()
+    /**
+     * Open {@link NeoStores} with all possible stores. If some store does not exist it will <b>not</b> be created.
+     * @return container with all opened stores
+     */
+    public NeoStores openNeoStores()
+    {
+        return openNeoStores( false, NeoStores.StoreType.values() );
+    }
+
+    /**
+     * Open {@link NeoStores} with all possible stores with a possibility to create store if it not exist.
+     * @param createStoreIfNotExists - should store be created if it's not exist
+     * @return container with all opened stores
+     */
+    public NeoStores openNeoStores(boolean createStoreIfNotExists)
+    {
+        return openNeoStores( createStoreIfNotExists, NeoStores.StoreType.values() );
+    }
```

Comment:
I would call these two `openAllNeoStores` to make sure that they are not accidentally confused with their varargs companions. At call-sites, they look like invocations of the varargs overloads, but with empty `storeTypes` arguments, which kind of have the opposite semantics in terms of which stores are opened.

Also you need spaces in the parameter lists of this method and the next.